### ISSUE TITLE
Linters

### ISSUE THREAD

**Issue by hfossli** [agensdev/playbook-ios#26]
I think we should just configure a linter and not having to specify where the colon delimiter goes in a readme. :D We could just agree about the config file and be done with the syntax.

In swift there's quite a few linters
- https://github.com/realm/SwiftLint
- https://github.com/Jintin/Swimat
- https://github.com/nicklockwood/SwiftFormat
- https://github.com/sleekbyte/tailor
- https://github.com/apple/swift/pull/3439 (currently only indentation)

In objc we have
- clang-format
- https://github.com/uncrustify/uncrustify

After having tested both clang-format and uncrustify I must say I found uncrustify to be the better choice (more options and easy to use).

I want our example project to have linters.

**Comment by kostiakoval** [comment]
I agree on having tools both linters and formatters, but I think we should also write in that format and think, not just blindly rely on tools.
